components/post_view/channel_intro_message/channel_intro_message.jsx

```diff
@@ -1,70 +1,95 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I reorganized the file in my local repository, and everything looks right. Anyway, I prefer in this kind of big PRs not include this kind of reorganization, because the PR is already complicated to review.